### PR TITLE
CP-40842: Remove CIS upload from Server Status Report wizard

### PR DESCRIPTION
**Pull request by danilo-delbusso**
Please review commits one by one.

### CODE REVIEW

File: XenAdmin/Commands/BugToolCommand.cs [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
 using XenAdmin.Network;
 using XenAdmin.Wizards;
+using XenAdmin.Wizards.BugToolWizardFiles;
```
Comment (CitrixChris) [inline XenAdmin/Commands/BugToolCommand.cs:34]: It looks like this is an unused import.

**Review by CitrixChris — APPROVED** [review]

**Review by CitrixChris — CHANGES_REQUESTED** [review]
Not sure  about the import but otherwise seems fine

File: XenAdmin/Commands/BugToolCommand.cs (hunk 1 above)
Comment (danilo-delbusso) [reply XenAdmin/Commands/BugToolCommand.cs:34]: well spotted. Also, there's quite a few, better to get rid of all of them 😎

**Comment by danilo-delbusso** [comment]
Force pushed to resolve conflict in `XenModel/Actions/AsyncAction.cs`

**Review by CitrixChris — APPROVED** [review]

**Comment by kc284** [comment]
This needs rebasing.
Also, when we're ready to merge I suggest we squash everything because it will need backporting to release/stockholm/lcm and one commit is much easier to handle.

**Comment by danilo-delbusso** [comment]
force pushed to resolve conflicts

**Review by CitrixChris — APPROVED** [review]

**Pull request merged**